### PR TITLE
Application style

### PR DESCRIPTION
**Pull request by CedrickFlocon**
Fix : https://github.com/home-assistant/home-assistant-android/issues/91 
https://github.com/home-assistant/home-assistant-android/issues/84

### CODE REVIEW

**Comment by CedrickFlocon** [comment]
Screenshot 2019-12-15 at 16 06 28

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -13,15 +13,15 @@
...
         <item name="android:textSize">20sp</item>
     </style>
 
-    <style name="TextAppearance.HomeAssistant.Title" parent="TextAppearance.AppCompat.Title">
+    <style name="TextAppearance.HomeAssistant.Title" parent="TextAppearance.MaterialComponents.Headline2">
         <item name="android:textSize">18sp</item>
     </style>
 
-    <style name="TextAppearance.HomeAssistant.Medium" parent="TextAppearance.AppCompat.Medium">
+    <style name="TextAppearance.HomeAssistant.Medium" parent="TextAppearance.MaterialComponents.Headline3">
```
Comment (gmatyszczak) [inline app/src/main/res/values/styles.xml]: There are also `Widget.HomeAssistant.Button.Colored`, `Widget.HomeAssistant.Button.Borderless` and `Theme.HomeAssistant.Dialog.Alert` in styles which extend from `AppCompat`. You should change them as well

Comment (CedrickFlocon) [reply app/src/main/res/values/styles.xml]: done

Comment (gmatyszczak) [reply app/src/main/res/values/styles.xml]: Looks good now :)

**Pull request merged**